canonical-web-and-design/juju.is
# Add the multi-cloud-ops about page

## QA
- Go to http://0.0.0.0:8041/about/multi-cloud-operations
- Check the content matches the [copy doc](https://docs.google.com/document/d/1NsFdr1yCMECGCA4qwLQGzDpQ6xYlLMRYbhOjW_EP9LU/edit#)

Fixes https://github.com/canonical-web-and-design/juju.is/issues/95
